SPRINT_MISSION_3/app.js

```diff
@@ -0,0 +1,25 @@
+import express from 'express';
+import dotenv from 'dotenv';
+
+import productRoutes from './routes/products.routes.js';
+import articleRoutes from './routes/Article.routes.js';
+import commentRouter from './routes/comment.routes.js';
+import errorHandler from './middlewares/errorHandler.js';
+import uploadRouter from './routes/upload.routes.js';
+
+dotenv.config();
+
+const app = express();
+app.use(express.json());
+
+// 라우터 등록
+app.use('/products', productRoutes);
+app.use('/article', articleRoutes);
+app.use('/api', commentRouter);
+app.use(errorHandler);
+app.use('/upload', uploadRouter);
```

Comment:
에러 핸들러는 모든 라우트를 등록하고 나서 등록해야해요.
```suggestion
app.use('/upload', uploadRouter);
app.use(errorHandler);
```